editor/code/package.json

```diff
@@ -170,9 +170,10 @@
   },
   "main": "./out/src/client.js",
   "scripts": {
-    "vscode:prepublish": "npm run esbuild -- --minify",
+    "vscode:prepublish": "npm run typecheck && npm run esbuild -- --minify",
```

Comment:
Actually I was failing to make this work properly, but I've seen where the problem is, we need to call `esbuild` instead of `node esbuild.mjs`.

Once sec, I push a PR.

Reply:
Actually I'm unsure how to best implement this :/